project/REVIEWING.md

```diff
@@ -186,6 +187,7 @@ to ease future classification:
  * `impact/cli` signifies the patch impacted a CLI command
  * `impact/dockerfile` signifies the patch impacted the Dockerfile syntax
  * `impact/deprecation` signifies the patch participates in deprecating an existing feature
+ * `impact/vendor` signifies the patch updated a vendored package
```

Comment:
Was thinking if this should be `impact/` or `area/`. Also see https://github.com/moby/moby/pull/46428, where we were discussing some of that.

`impact/` tends to be "important changes, potentially affecting end-users". I guess "end-users" is a bit of a grey-area (consumers of our code could be considered an end-user), but for users of the binaries, this may be less "impactful".

Reply:
I went with `impact/` mostly because `area/` also apply to the issues (and are defined in [`ISSUE_TRIAGE.md`](https://github.com/moby/moby/blob/7da33a5e92ceea671daab436dd5849d9de250216/project/ISSUE-TRIAGE.md)).

But probably that's just a "soft convention" and we could make it an `area/`. I'm fine with both options.

Reply:
Yeah, thinking there was that we could use `area/` to group items in the release-notes (have a "dependency changes" section like containerd has); https://github.com/containerd/containerd/releases/tag/v1.7.13

We can discuss in the maintainers call if we want. This is really bike-shedding of course 🙈